packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -140,7 +141,11 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         return this.dataStoreContext.IFluidHandleContext;
     }
 
-    private readonly serializer = new FluidSerializer(this.IFluidHandleContext);
+    public referenceAdded(sourcePath: string, referencedHandle: IFluidHandle): void {
+        (this.routeContext as any).referenceAdded(sourcePath, referencedHandle);
```

Comment:
The cast to any is temporary here. Once this API shape looks good, I will make the change in definitions first and then update the code here accordingly.